build.gradle

```diff
@@ -352,6 +352,32 @@ subprojects {
     }
   }
 
+  jacocoTestReport {
+    dependsOn test // tests are required to run before generating the report
+  }
+
+  afterEvaluate {
+    jacocoTestCoverageVerification {
+      dependsOn jacocoTestReport
+
+      violationRules {
+        rule {
+          def threshold = project.ext.has('jacocoCoverageThreshold') ?
```

Comment:
Is the intend to remove this customization once all modules are sufficiently covered?

Reply:
Yes. Ideally, we want to use one coverage, i.e. 0.8, for all modules, but that's a long way to go